spring-boot-project/spring-boot/src/test/java/org/springframework/boot/web/embedded/tomcat/TomcatReactiveWebServerFactoryTests.java

```diff
@@ -59,13 +59,14 @@ protected TomcatReactiveWebServerFactory getFactory() {
 	@Test
 	public void tomcatCustomizers() {
 		TomcatReactiveWebServerFactory factory = getFactory();
-		TomcatContextCustomizer[] listeners = new TomcatContextCustomizer[4];
-		Arrays.setAll(listeners, (i) -> mock(TomcatContextCustomizer.class));
-		factory.setTomcatContextCustomizers(Arrays.asList(listeners[0], listeners[1]));
-		factory.addContextCustomizers(listeners[2], listeners[3]);
+		TomcatContextCustomizer[] customizers = new TomcatContextCustomizer[4];
+		Arrays.setAll(customizers, (i) -> mock(TomcatContextCustomizer.class));
+		factory.setTomcatContextCustomizers(
+				Arrays.asList(customizers[0], customizers[1]));
+		factory.addContextCustomizers(customizers[2], customizers[3]);
 		this.webServer = factory.getWebServer(mock(HttpHandler.class));
-		InOrder ordered = inOrder((Object[]) listeners);
-		for (TomcatContextCustomizer listener : listeners) {
+		InOrder ordered = inOrder((Object[]) customizers);
+		for (TomcatContextCustomizer listener : customizers) {
```

Comment:
Can you please rename `listener` to `customizer` there as well?

Reply:
@snicoll Oops, I missed them somehow 😅 Thanks for catching them 👍 I've updated and force-pushed.